Add tab "Active List"

ykuzakova opened the pull request.

ykuzakova committed:
Add tab "Active List"

ykuzakova committed:
Add tab "Active List"

ykuzakova committed:
Add functionality in active list

ykuzakova committed:
Change toolbar item status

ykuzakova committed:
Save activelist tab after Service1Action

ykuzakova committed:
change service's status

ykuzakova committed:
change icon start service in started state

sibprogrammer commented on meta.xml:
```diff
@@ -8,5 +8,5 @@
   <release>5</release>
   <vendor>Plesk</vendor>
-  <plesk_min_version>12.5</plesk_min_version>
+  <plesk_min_version>17.0</plesk_min_version>
```
let's make it possible to work both on Plesk 12.5 & 17.0

sibprogrammer commented on plib/controllers/IndexController.php:
```diff
@@ -239,4 +243,8 @@ public function removeAction()
         $this->_helper->json(['status' => 'success', 'statusMessages' => $messages]);
     }
+
+    public function activelistAction()
```
capitalize List word

sibprogrammer commented on plib/resources/locales/en-US.php:
```diff
@@ -1,5 +1,18 @@
 <?php
 // Copyright 1999-2016. Parallels IP Holdings GmbH.
-$messages = [
+$messages = array(
```
array short syntax definition should be used

sibprogrammer commented on plib/views/scripts/index/activelist.phtml:
```diff
@@ -0,0 +1,112 @@
+?>
+<?php echo $this->renderTabs($this->tabs); ?>
+
+<h3>Your Containers</h3>
```
missed localization

sibprogrammer commented on plib/views/scripts/index/activelist.phtml:
```diff
@@ -0,0 +1,112 @@
+                ],
+            ],
+
+            'additionalHtml' => '<H3>Some html block for data<H3>',
```
missed localization

sibprogrammer commented on plib/views/scripts/index/activelist.phtml:
```diff
@@ -0,0 +1,112 @@
+                    'link' => pm_Context::getActionUrl('index', 'form') . '/id/form',
+                    'toolbar' => [
+                        [
+                            'title' => 'toolbar in service',
```
missed localization

sibprogrammer commented on plib/views/scripts/index/activelist.phtml:
```diff
@@ -0,0 +1,112 @@
+
+    ],
+
+]); ?>
```
closing PHP tag is prohibited for files that contain only PHP code

sibprogrammer commented on plib/controllers/IndexController.php:
```diff
@@ -23,6 +23,10 @@ public function init()
                 'action' => 'list',
             ],
+            [
+                'title' => 'Active List',
```
seems we forgot about localization

ykuzakova committed:
Fix pull request #3:
 + Add some localization
 + Capitalize list to List (activelist -> activeList)
 + Remove PHP closing tag

ykuzakova committed:
Add activity to "Start/Stop Service" button and show "Status Active/Inactive"

sibprogrammer commented on meta.xml:
```diff
@@ -9,4 +9,5 @@
   <release>5</release>
   <vendor>Plesk</vendor>
   <plesk_min_version>12.5</plesk_min_version>
+  <plesk_max_version>17.0</plesk_max_version>
```
This leads to inability to use the extension on Plesk 17.0.

sibprogrammer commented on plib/controllers/IndexController.php:
```diff
@@ -12,17 +12,21 @@ public function init()
+                'title' => pm_Locale::lmsg('listTitle'),
                 'action' => 'list',
             ],
+            [
```
ActiveList is absent in Plesk 12.5. So we shouldn't add the tab with ActiveList in case of Plesk 12.5. Use "if (plesk version >= 17)" conditional clause. See "ext-custom-button" extension for example.

ykuzakova committed:
Vew 'ActiveList' tab for Plesk versions greater or equal 17.0.
